src/ace-adapter.ts

```diff
@@ -0,0 +1,318 @@
+import { Ace, Range } from 'ace-builds';
+
+import {
+  ClientIDType,
+  EditorAdapterEvent,
+  EditorEventCallbackType,
+  IEditorAdapter,
+  IEditorAdapterEvent,
+  UndoRedoCallbackType,
+} from "./editor-adapter";
+import { EventListenerType } from './emitter';
+import { Cursor, ICursor } from './cursor';
+import { IDisposable } from './utils';
+import { ITextOperation, TextOperation } from './text-operation';
+
+export class ACEAdapter implements IEditorAdapter {
+  ignoreChanges = false;
+  private ace: Ace.Editor;
```

Comment:
No consistency in class member declaration (accessibility wise).

For data members we have public members followed by private ones.
For methods we have public followed by private members and then again public methods.

Reply:
updated member ordering public methods -> private methods